### ISSUE TITLE
FunctionToTransformEachPageLink causes duplicate page links to be rendered

### ISSUE THREAD

**Issue by ajbeaven** [dncuug/X.PagedList#201]
**Describe the bug**
When FunctionToTransformEachPageLink  is specified in the options, it appears like duplicate page links are being rendered when calling Html.PagedListPager. Removing FunctionToTransformEachPageLink from the options makes the pages render as expected.

Using X.PagedList.Mvc.Core v8.0.7 on a .NET 5 project.

**To Reproduce**
1. Use options with FunctionToTransformEachPageLink rendering the list page eg:
```
@Html.PagedListPager(
	Model.Items,
	page => Url.Action("Index", "Items", Context.Request.Query.ToRouteValues(new { page, area = "" })),
	new PagedListRenderOptions
	{
		FunctionToTransformEachPageLink = (li, a) =>
		{
			li.InnerHtml.AppendHtml(a);
			return li;
		}
	})
```

**Expected behavior**
Only single set of pages should be rendered

**Screenshots**
![image](https://user-images.githubusercontent.com/657201/106682444-63429e00-6627-11eb-8517-5581b29220be.png)

**Desktop**
 - OS: Windows 10
 - Browser Edge
 - Version 88

**Comment by ajbeaven** [comment]
Looks like this is fixed in the latest code - just waiting for a new nuget release.

**Comment by ajbeaven** [comment]
The recently released nuget package 8.1 fixes this. Thanks!
